locales/fr.yaml

```diff
@@ -144,10 +144,10 @@ buyer_took_your_order: |
   @${buyerUsername} a récupéré ton offre et veut acheter tes sats. Contacte-le pour lui indiquer comment t'envoyer ${fiatAmount} ${currency} via ${paymentMethod}.
 
   Dès que tu as vérifié que le paiement est arrivé en totalité, tu dois débloquer les sats
-waiting_seller_to_pay: "J'ai envoyé une requête de paiement au vendeur pour qu'il envoie tes sats concernant l'offre dont la référence est #${orderId}, dès que le paiement est fait je vous informerai tous les deux."
+waiting_seller_to_pay: J'ai envoyé une requête de paiement au vendeur pour qu'il envoie tes sats concernant l'offre dont la référence est #${orderId}, dès que le paiement est fait je vous informerai tous les deux.
```

Comment:
did you tested this? as J'ai have a `'` we should wrapped it on quotes

Reply:
How to test?

And how to wrap it ? Plz give an exemple as there is many other occurences of this case in the file that i should fix then.

Reply:
to test you need to set a local environment for the bot, it's a bit technical, I what I'm trying to test you is that line will fail, please put it back as it was